arrays/Arrays.jsh

```diff
@@ -0,0 +1,13 @@
+void s(long time){
+	try{
+		Thread.sleep(time);
+	} catch (InterruptedException e){
+	}
```

Comment:
serr przynajmniej mogłoby tu być. Jest coś takiego jak spurious wakeup, co może wybudzić uśpiony wątek spoza aplikacji zatem nawet jak Ty nie planujesz go budzić lub mu przerywać, to się może zdarzyć.